stories/topics/con-aws-application-architecture.adoc

```diff
@@ -10,17 +10,9 @@ image::aap-on-aws-architecture.png[AWS application architecture]
-Components deployed are isolated into their own _Virtual Private Cloud_ (VPC). 
-Applications run on instances backed by an autoscale group, which run on RHEL-based _Amazon Machine Images_ (AMIs) as their basis. 
-These AMIs are preloaded with all the required container images to run the {PlatformNameShort}, {HubName}, {ControllerName}, and {ReceptorRPM} components. 
-A shared EFS volume is mounted into each instance stood up by the autoscale groups.
-
-All internal instances have private IP addresses by default and internal traffic is encrypted by default through {AWS}. 
-The {PlatformNameShort} application is run in containers managed through podman on the instances provisioned by the autoscale groups. 
-This podman configuration is itself managed by systemd. 
-SELinux is enabled on each instance and is supported down to the container level. 
-Each {ControllerName} instance is configured as an {PlatformNameShort} hybrid node by default to simplify the scale up and out procedures.
+{AAPonAWS}  is designed to be private, with no public access allowed by default. This requires customers to expose the deployed internal Application Load Balancers (ALBs) themselves pursuant to their own network requirements and security practices (some potential ways to expose the ALBs would include VPC Peering, Transit Gateway, VPN, External Load Balancers, etc...). All cloud infrastructure components are deployed in a Virtual Private Cloud (VPC). Customers may choose between deploying into an existing VPC or they may opt to have the product deploy a new VPC for them.  All VM instances and Cloud infrastructure have private IP addresses (allocation determined by the VPC and subnetworks specified at deployment time) by default. All internal traffic is encrypted using self-signed certificates generated at deployment time (external traffic may also be encrypted by deploying your own certificate on the Application Load Balancers deployed by the product). The {PlatformNameShort} software runs as containers on the deployed VM instances.  VM instance lifecycles are managed by Autoscaling Groups (ASG) that monitor the health of each service running on the VM instances and will automatically cycle the VM instances down and replace them with new VM instances if the health check fails to respond ensuring that the {PlatformNameShort} services stay up and available to process requests. The VM instances run a customized RHEL (RedHat Enterprise Linux) Amazon Machine Image (AMI) as their base image. This AMI is preloaded with all the required container images and packages to run the {PlatformNameShort} ({HubName}, {ControllerName}, and Execution Node components). A shared EFS (Elastic File Store) volume is mounted into each VM instance provisioned by the product and is utilized for shared access to common files and resources.  A Relational Database Service (RDS) is provisioned by the product at deployment time and contains databases for both the {ControllerName} and {HubName}.  
```

Comment:
_Application Load Balancers_ (ALBs)
practices. Some potential ways to include the ALBs include .....  Balancers, amongst others.
Customers can choose between deploying into an existing VPC or canchoose to have the product deploy a new VPC for them.
(external traffic can also...
_Autoscaling Groups_ (ASGs) manage VM instances and monitor the the health...
VM instances. ASGs automatically cycle...